src/components/common/graph/d3-graph-elevation.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, it, expect, beforeEach, afterEach } from 'vitest'
+import { mount } from '@vue/test-utils'
+
+import D3GraphElevation from './d3-graph-elevation.vue'
+
+describe('D3GraphElevation', () => {
+  let wrapper: ReturnType<typeof mount>
+
+  beforeEach(() => {
+    wrapper = mount(D3GraphElevation, {
+      props: {
+        showTooltip: true,
+      },
+    })
+  })
+
+  afterEach(() => {
+    wrapper.unmount()
+  })
+
+  it('renders properly with default props', () => {
```

Comment:
does any console error in tests make the test fail ?
If not, is there a way to check that no error in rendering has occurred ?

Reply:
> does any console error in tests make the test fail ? 
I don't understand, do you mean that the unit test should catch the console errors if any? I think it can't, plus this is not unit tests are made for.

> If not, is there a way to check that no error in rendering has occurred ?
Checking if the UI result is rendered correctly is not the unit test job. Of course we can test the rendering with e2e tests, but I won't do it now because the d3 graph uses temporary fixtures.